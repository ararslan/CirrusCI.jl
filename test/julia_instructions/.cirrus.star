load("../../lib.star", "julia_instructions")
load("github.com/cirrus-modules/helpers", "task", "freebsd_instance")

def main(ctx):
    return [task("No coverage", freebsd_instance("freebsd-13-0"),
                 instructions=julia_instructions("FreeBSD")),
            task("Codecov", freebsd_instance("freebsd-13-0"),
                 instructions=julia_instructions("FreeBSD", coverage="codecov"))]
