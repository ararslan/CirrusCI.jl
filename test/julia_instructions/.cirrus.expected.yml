task:
  name: No coverage
  freebsd_instance:
    image_family: freebsd-13-0
  install_script:
    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
  build_script:
    - cirrusjl build
  test_script:
    - cirrusjl test
task:
  name: Codecov
  freebsd_instance:
    image_family: freebsd-13-0
  install_script:
    - sh -c "$(fetch https://raw.githubusercontent.com/ararslan/CirrusCI.jl/master/bin/install.sh -o -)"
  build_script:
    - cirrusjl build
  test_script:
    - cirrusjl test
  coverage_script:
    - cirrusjl coverage codecov
